### ISSUE TITLE
Code completion only showing classes

### ISSUE THREAD

**Issue by rahularyan** [HvyIndustries/crane#360]
Platform : win32
VSCode Version : 1.16.0
Crane Version : 0.3.8
---

Hello,
I am facing a weird issue since I moved from VS code insiders to VS code. Code completion only showing classes suggestion never shows any function. I tried to recompile many time but same thing happens always. As I remember it was working in insider version.
Is this issue have something to do with PHP namespace? I recently started added namespaces to my project (only in few files) but not sure if it was work before adding namespace.

Lemme know if you need more info. Thanks

**Comment by nevadascout** [comment]
When you say function, do you mean "top level" functions in a file, or functions on a class (methods)?

**Comment by rahularyan** [comment]
Yes I mean function not method. Though method are not showing as well.

Sent from Outlook for Android

________________________________

**Comment by DividedByCero** [comment]
I declared a functions outside of a class within a namespace too, but when i get the function in the autocompletion, it doesn't refer to the namespace where was declared, instead, it's appearing as a function owned by the file where i'm trying to call it, and then when I hit enter, i get `Function()` instead of `Namespace\SubNamespace\Function()` or `Namespace\Function()`. I was thinking that i was doing  something wrong declaring my functions.

**Comment by nevadascout** [comment]
Thanks @DividedByCero, this is a separate bug.

@rahularyan Methods should be working, have you tried uninstalling and re-installing the plugin?

**Comment by rahularyan** [comment]
Yes but that did nothing.
For now I am switching to PHP IntelliSense by Felix Beacker which does not work well with classes and methods. I think installing both PHP IntelliSense plugin will solve our issue (not tried, may be it will end up showing duplicate results :) ).
Lemme know if you need any further clarification on this bug.
